platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ImportCommand.java

```diff
@@ -54,18 +63,53 @@ public ImportCommand() {}
       Security security,
       EventAdmin eventAdmin,
       Session session,
-      boolean profile) {
+      boolean profile,
+      boolean force) {
     super(service, security, eventAdmin, session);
     this.profile = profile;
+    this.force = force;
   }
 
   @Override
   protected Object executeWithSubject() {
+    if (isAccidentalImport()) {
```

Comment:
Would it be better to flip the logic on this statement, and include lines 78-82 in the if condition. That way there would only be one `return`

Reply:
Additionally, you could have the statement check for `force` here to avoid calling `isAccidentalImport()`